support-frontend/assets/helpers/redux/checkout/addressMeta/reducer.ts

```diff
@@ -17,11 +17,8 @@ export const addressMetaSlice = createSlice({
 		setDeliveryAgent(state, action: PayloadAction<number | undefined>) {
 			state.deliveryAgent.chosenAgent = action.payload;
 		},
-		setDeliveryAgentResponse(
-			state,
-			action: PayloadAction<DeliveryAgentsResponse | undefined>,
-		) {
-			state.deliveryAgent.response = action.payload;
+		clearDeliveryAgentResponse(state) {
```

Comment:
For the record: We renamed this to make it clearer what it’s being used for, because we initially added code here instead of in the “thunk fulfilled” reducer below.